mscore/workspace.cpp

```diff
@@ -152,14 +152,25 @@ void MuseScore::deleteWorkspace()
             return;
       preferences.dirty = true;
       Workspace* workspace = 0;
-      foreach(Workspace* p, Workspace::workspaces()) {
+      for (Workspace* p : Workspace::workspaces()) {
             if (p->name() == a->text()) { // no need for qApp->translate since "Basic" and "Advanced" are not deletable
                   workspace = p;
                   break;
                   }
             }
       if (!workspace)
             return;
+
+      QMessageBox::StandardButton reply;
+      reply = QMessageBox::question(0,
+                 QWidget::tr("Warning"),
+                 QWidget::tr("Are you sure?"),
```

Comment:
We probably need a better message and title no? If you let your computer on this dialog for a while and come back: Are you sure about what?